app/components/channel_list/categories/body/channel/channel_list_item.tsx

```diff
@@ -61,28 +61,30 @@ const textStyle = StyleSheet.create({
 });
 
 type Props = {
-    channel: Pick<ChannelModel, 'deleteAt' | 'displayName' | 'name' | 'shared' | 'type'>;
+    channel: ChannelModel;
     isActive: boolean;
-    isOwnDirectMessage: boolean;
     isMuted: boolean;
     myChannel?: MyChannelModel;
     collapsed: boolean;
+    currentUserId: string;
 }
 
-const ChannelListItem = ({channel, isActive, isOwnDirectMessage, isMuted, myChannel, collapsed}: Props) => {
+const ChannelListItem = ({channel, isActive, currentUserId, isMuted, myChannel, collapsed}: Props) => {
     const {formatMessage} = useIntl();
     const theme = useTheme();
     const styles = getStyleSheet(theme);
     const serverUrl = useServerUrl();
 
+    const isOwnDirectMessage = (channel.type === General.DM_CHANNEL) && currentUserId === getUserIdFromChannelName(currentUserId, channel.name);
```

Comment:
We removed this from the withObservables to reduce the number of pipes. Not sure if needed, but I think the code is more readable.